### PR TITLE
MGMT-3854 log progress new API

### PR DESCRIPTION
**Pull request by slaviered**
- code review changes:
   - Separate API for host and cluster
   - Progress data should be embedded in json within the request body

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -2000,6 +2000,65 @@ paths:
...
+      parameters:
+        - in: path
+          name: cluster_id
+          description: The cluster whose log progress is being updated.
+          type: string
+          format: uuid
+          required: true
+        - in: path
+          name: host_id
+          description: The host whose logs should be uploaded.
```
Comment (avishayt) [inline swagger.yaml:2020]:
```suggestion
          description: The host whose logs status be updated.
```

Comment (slaviered) [reply swagger.yaml:2020]: We use the term uploaded all over the code

**Review by avishayt — APPROVED** [review]

**Pull request merged**